cmd/mattermost/commands/user_test.go

```diff
@@ -84,16 +81,14 @@ func TestChangeUserEmail(t *testing.T) {
 	newEmail := model.NewId() + "@mattermost-test.com"
 
 	th.CheckCommand(t, "user", "email", th.BasicUser.Username, newEmail)
-	if _, err := th.App.Srv.Store.User().GetByEmail(th.BasicUser.Email); err == nil {
-		t.Fatal("should've updated to the new email")
-	}
-	if user, err := th.App.Srv.Store.User().GetByEmail(newEmail); err != nil {
-		t.Fatal(err)
-	} else {
-		if user.Email != newEmail {
-			t.Fatal("should've updated to the new email")
-		}
-	}
+	_, err := th.App.Srv.Store.User().GetByEmail(th.BasicUser.Email)
```

Comment:
Hi @Lumexralph I was just going through your pull request and trying to understand how `err` has been initialized twice on line no. 84 and 87?

Reply:
@goku321 I appreciate your feedback. a new variable `user` was created in line 87 i.e `user, err := ..` compared to line 84. In Go, even if there was an already declared variable, for the fact that a new variable is created when assigning new values e.g `user`, we can redeclare `err` this way. I hope this helps your question?

For more reference, please check this out https://golang.org/ref/spec#Short_variable_declarations.

Reply:
Thanks @Lumexralph for explaining. It is really helpful :)